webpack.dev.config.js

```diff
@@ -0,0 +1,11 @@
+const { createConfig } = require('@edx/frontend-build');
+
+const config = createConfig('webpack-dev', {
+  devServer: {
+    historyApiFallback: {
+      disableDotRule: true,
```

Comment:
Allows the use of `.` in URLs when running the MFE using the webpack dev server. Without `disableDotRule: true`, the URL `http://localhost:8734/test-enterprise/course/edX+CTL.SC2x` won't load, resulting in a `Cannot GET <path>`.

Reply:
Will be replaced by https://github.com/openedx/frontend-build/pull/352